apps/e2e/lookup-storybook/src/app/lookup/lookup.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, Input } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup } from '@angular/forms';
+import { SkyAutocompleteSearchFunctionFilter } from '@skyux/lookup';
+
+export interface Names {
+  name: string;
+}
+@Component({
+  selector: 'app-lookup',
+  templateUrl: './lookup.component.html',
+  styleUrls: ['./lookup.component.scss'],
+})
+export class LookupComponent {
+  @Input()
+  public selectMode: 'single' | 'multiple' = 'single';
+
+  @Input()
+  public disabledFlag = false;
+
+  public placeholderText = 'This is what placeholder text looks like';
+
+  public favoritesForm: FormGroup<{
+    favoriteNames: FormControl<Names[] | null>;
+    favoriteNamesAll: FormControl<Names[] | null>;
+    favoriteNamesFew: FormControl<Names[] | null>;
+  }>;
+
+  public searchFilters: SkyAutocompleteSearchFunctionFilter[] = [];
```

Comment:
`searchFilters` is not being used anywhere and can be removed here and in the HTML everywhere.